GCDWebUploader/GCDWebUploader.m

```diff
@@ -296,7 +296,13 @@ @implementation GCDWebUploader
 
 - (instancetype)initWithUploadDirectory:(NSString*)path {
   if ((self = [super init])) {
-    NSBundle* siteBundle = [NSBundle bundleWithPath:[[NSBundle mainBundle] pathForResource:@"GCDWebUploader" ofType:@"bundle"]];
+    NSString *bundlePath = [[NSBundle mainBundle] pathForResource:@"GCDWebUploader"
+                                                           ofType:@"bundle"];
+    if (!bundlePath) {
```

Comment:
You don't need this as `[[NSBundle bundleForClass:[GCDWebUploader class]] ` should work in both cases.